waiter/integration/waiter/basic_test.clj

```diff
@@ -178,6 +179,27 @@
       (is (= 400 status))
       (is (str/includes? body "Command type fakecommand is not supported")))))
 
+(deftest ^:parallel ^:integration-fast test-basic-parameters-exceed-limits
```

Comment:
Can we be consistent on naming limits vs. constraints?  Should this be called `test-basic-parameters-outside-constraints`

Reply:
Renamed to `test-basic-parameters-violates-upper-limit-constraints`

Reply:
`test-basic-parameters-violates-max-constraints`